### DISCUSSION TITLE
Error on EdgeTX companion 2.8.0

### DISCUSSION THREAD

**Discussion by RCAviation** [EdgeTX/edgetx#3193] [Q&A]
Hello,

Can anyone please tell me what I'm doing wrong here. 

I installed Companion 2.8.0 but when I try to "Read Firmware from Radio I get a error message. I really have no idea how or where to search to solve the problem.

![image](https://user-images.githubusercontent.com/121538999/218860978-17306987-e49d-4e3e-8ef1-f9b322e8cb5a.jpeg)

Thanks in advance. 
Greetings from Marc, Holland NL

**Comment by gagarinlg** [comment]
There is a broken Model on the sd card. Open the model on the radio, change something, change it back and try again.

**Reply by RCAviation** [reply]
Thanks for the comment!
Strangely I only have two models yet. One for Flight sim via Dsmx usb transmitter and one for the plane I'm building right now. Can it be one of my "models" then?

**Comment by gagarinlg** [comment]
That is one that was put on the by the manufacturer.
If you don't need it, just rename the file by changing the extension. You should be good then.

**Reply by RCAviation** [reply]
Ok I will try tomorrow! Thanks!

**Comment by pfeerick** [comment]
model8 is most likely the "Eachine P-51D" one... you can delete it if you don't want it, or you can open it on the transmitter and make some simple change to it like changing a timer setting, and the invalid configuration will be corrected. c.f. https://github.com/EdgeTX/edgetx/issues/1969#issuecomment-1196288284

**Reply by RCAviation** [reply]
Hi,

I just thrown away the P-51 and finally Companion works. Now I can start learning companion, I can't wait but first I have to work for too many hours 🤪. Thanks for your help! Greetz 🇳🇱
